### ISSUE TITLE
What is low resolution image in context of semantic segmentation

### ISSUE THREAD

**Issue by shkarupa-alex** [wuhuikai/DeepGuidedFilter#34]
In semantic segmentation the default setup is:
- input image
- backbone which returns features at strides 2, 4, 8, 16, 32
- decoder which combines features and produces logits with stride-2 (or 4 as in DeepLabV3+)
- head which resizes logits to image size and makes predictions (conv+softmax/sigmoid)

So what should be a low resolution image for ConvGuidedFilter in this case?
Should i resize original image to logits size (stride 2 or 4) or i should use stride-2/4 features from backbone?

**Comment by wuhuikai** [comment]
Guidance image: img with HxW
Low-res input: img with H/4xW/4
Low-res output: logits with H/4 xW/4

**Comment by shkarupa-alex** [comment]
Low-res image obtained by just resizing original one?
Did you test what interpolation (linear/cubic/area) works better?

**Comment by wuhuikai** [comment]
Yes, I only tried bilinear.
